README.md

```diff
@@ -113,8 +109,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
     * [App Service](https://azure.microsoft.com/services/app-service/) - 10 web, mobile or API apps (60 CPU minutes / day)
     * [Functions](https://azure.microsoft.com/services/functions/) - 1 million requests per month
     * [DevTest Labs](https://azure.microsoft.com/services/devtest-lab/) - Enable fast, easy, and lean dev-test environments
-    * [Active Directory](https://azure.microsoft.com/services/active-directory/) - 500,000 objects
-    * [Active Directory B2C](https://azure.microsoft.com/services/active-directory/external-identities/b2c/) - 50,000 monthly stored users
```

Comment:
<img width="1131" alt="Screenshot 2023-04-16 at 14 58 07" src="https://user-images.githubusercontent.com/82342/232313018-b5402149-af14-433d-90cf-85a39910bb9e.png">

Reply:
Yes the free tier is still there might have removed it by mistake. Sorry.